src/components/cards/Cards.js

```diff
@@ -1,20 +1,48 @@
+import { useEffect, useState } from 'react'
 import Card from './card/Card'
 import './Cards.css'
-import { getDataArr } from './getDataArr'
 
 const Cards = props => {
-  let dataArr = getDataArr()
-  console.log('Cards-dataArr is of type: ' + typeof dataArr)
+  const [dataArr, setDataArr] = useState(null)
+  const [loading, setLoading] = useState(true)
+  const [error, setError] = useState(null)
 
-  if (props.house !== 'all') {
-    dataArr = dataArr.filter(
-      person => person.house.toLowerCase() === props.house
-    )
+  useEffect(() => {
+    fetch('http://hp-api.herokuapp.com/api/characters')
```

Comment:
Also, I think you'll need to set this URL to this:
```suggestion
    fetch('https://hp-api.herokuapp.com/api/characters')
```

Reply:
Changed it, even so in their documentation they use the http version

Reply:
Yea, I know - my bad though! Should have told you guys earlier 🙈